DetectorDescription/DDCMS/src/DDFilteredView.cc

```diff
@@ -194,13 +213,104 @@ bool DDFilteredView::firstChild() {
   while ((node = it_.back().Next())) {
     if (accept(noNamespace(node->GetVolume()->GetName()))) {
       node_ = node;
+      startLevel_ = it_.back().GetLevel();
       return true;
     }
   }
   LogVerbatim("DDFilteredView") << "Search for first child failed.";
   return false;
 }
 
+int DDFilteredView::nodeCopyNo(const std::string_view copyNo) const {
+  int result;
+  if (auto [p, ec] = std::from_chars(copyNo.data(), copyNo.data() + copyNo.size(), result); ec == std::errc()) {
+    return result;
+  }
```

Comment:
Is this incredibly obscure C++17 code really the recommended way to do `atoi()`?

Reply:
yep, it is, because `from_chars` is a low-level and offers the best possible performance.